pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -173,9 +173,11 @@ def archive_pdf_file(pdf_archive_path, binary_content, extract_as_dict):
     def get_wms_url_params():
         """
         Returns the list of additionally configured wms_url_params.
+        Args:
+            params: optional argument used for given custom parameters. Expecting a dict
```

Comment:
```suggestion
            params (dict): Optional argument used for given custom parameters.
```

Reply:
Thanks @ger-benjamin I removed the `params` it should not be there at all and changed the return value to meet the other descriptions.